lib/puppet/settings.rb

```diff
@@ -1294,7 +1294,7 @@ def lookup(name)
     # @param name [Symbol] The configuration setting name to look up
     # @return [Object] The configuration setting value or nil if the setting is not known
     # @api public
-    def interpolate(name)
+    def interpolate(name, to_string=false)
```

Comment:
As you've added a new param here, it should get added to the yarddoc above the method.